lib/Minz/Extension.php

```diff
@@ -360,28 +362,28 @@ private function removeConfiguration(string $type): void {
 		FreshRSS_Context::$$conf->save();
 	}
 
-	public final function removeSystemConfiguration(): void {
+	final public function removeSystemConfiguration(): void {
 		$this->removeConfiguration('system');
 		$this->system_configuration = null;
 	}
 
-	public final function removeUserConfiguration(): void {
+	final public function removeUserConfiguration(): void {
 		$this->removeConfiguration('user');
 		$this->user_configuration = null;
 	}
 
-	public final function saveFile(string $filename, string $content): void {
+	final public function saveFile(string $filename, string $content): void {
 		$username = Minz_User::name();
 		$path = USERS_PATH . "/{$username}/{$this->config_key}/{$this->getName()}";
 
-		if (!file_exists($path)) {
-			mkdir($path, 0777, true);
+		if (!file_exists($path) && !mkdir($path, 0777, true) && !is_dir($path)) {
+			throw new RuntimeException(sprintf('Directory "%s" was not created', $path));
```

Comment:
Have you tested the behaviour, both with and without `'environment' => 'development',` in `config.php`?
What is the outcome?